locations/spiders/dennys.py

```diff
@@ -12,56 +22,80 @@ class DennysSpider(scrapy.Spider):
         'https://locations.dennys.com/',
     )
 
-    def store_hours(self, store_hours):
-        day_groups = []
-        this_day_group = None
-        for day_info in store_hours:
-            day = day_info['day'][:2].title()
-
-            hour_intervals = []
-            for interval in day_info['intervals']:
-                f_time = str(interval['start']).zfill(4)
-                t_time = str(interval['end']).zfill(4)
-                hour_intervals.append('{}:{}-{}:{}'.format(
-                    f_time[0:2],
-                    f_time[2:4],
-                    t_time[0:2],
-                    t_time[2:4],
-                ))
-            hours = ','.join(hour_intervals)
-
-            if not this_day_group:
-                this_day_group = {
-                    'from_day': day,
-                    'to_day': day,
-                    'hours': hours
-                }
-            elif this_day_group['hours'] != hours:
-                day_groups.append(this_day_group)
-                this_day_group = {
-                    'from_day': day,
-                    'to_day': day,
-                    'hours': hours
-                }
-            elif this_day_group['hours'] == hours:
-                this_day_group['to_day'] = day
-
-        day_groups.append(this_day_group)
-
-        opening_hours = ""
-        if len(day_groups) == 1 and day_groups[0]['hours'] in ('00:00-23:59', '00:00-00:00'):
-            opening_hours = '24/7'
-        else:
-            for day_group in day_groups:
-                if day_group['from_day'] == day_group['to_day']:
-                    opening_hours += '{from_day} {hours}; '.format(**day_group)
-                elif day_group['from_day'] == 'Su' and day_group['to_day'] == 'Sa':
-                    opening_hours += '{hours}; '.format(**day_group)
-                else:
-                    opening_hours += '{from_day}-{to_day} {hours}; '.format(**day_group)
-            opening_hours = opening_hours[:-2]
-
-        return opening_hours
+    # def store_hours(self, store_hours):
```

Comment:
If this code isn't needed anymore, let's delete it instead of commenting it out.

Reply:
Oh oops, yeah I meant to delete that stuff after testing. Will fix.